vispy/visuals/graphs/layouts/__init__.py

```diff
@@ -0,0 +1,52 @@
+import inspect
+
+from .random import random
+from .circular import circular
+from .force_directed import fruchterman_reingold
+
+
+_layout_map = {
+    'random': random,
+    'circular': circular,
+    'force_directed': fruchterman_reingold,
+    'spring_layout': fruchterman_reingold
+}
+
+AVAILABLE_LAYOUTS = _layout_map.keys()
+
+
+def get(name, *args, **kwargs):
```

Comment:
I think users will use this too, because it also allows to configure some layouts (for example setting the initial node positions with the Fruchterman-Reingold algorithm)